app/controllers/chat_channel_memberships_controller.rb

```diff
@@ -26,6 +26,8 @@ def edit
   def create
     membership_params = params[:chat_channel_membership]
     @chat_channel = ChatChannel.find(membership_params[:chat_channel_id])
+    raise Pundit::NotAuthorizedError if @chat_channel.private_org_channel? # Not available for invite.
```

Comment:
Can we make this part of the existing policy that we authorize on the next line? Seems if we add `!record.private_org_channel?` to `user_can_edit_channel` it would work too?